### PR TITLE
build(examples): switch TodoMVC main to webpack

### PR DESCRIPTION
**Pull request by KingHenne**

### CODE REVIEW

File: examples/todomvc-main/tsconfig.build.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
+    "declaration": false,
```
Comment (clebert) [inline examples/todomvc-main/tsconfig.build.json]: please move to tsconfig.json

File: examples/todomvc-main/webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+// @ts-check
+const path = require('path');
```
Comment (clebert) [inline examples/todomvc-main/webpack.config.js]: newline

File: examples/todomvc-main/tsconfig.build.json (hunk 1 above)
Comment (KingHenne) [reply examples/todomvc-main/tsconfig.build.json]: done

File: examples/todomvc-main/webpack.config.js (hunk 2 above)
Comment (KingHenne) [reply examples/todomvc-main/webpack.config.js]: done

**Pull request merged**